app/controllers/sessions_controller.rb

```diff
@@ -21,8 +21,8 @@ def create_vimeo
     auth = request.env["omniauth.auth"]
     au_user = AuUser.find_by_provider_and_uid(auth["provider"], auth["uid"]) || AuUser.create_with_omniauth(auth)
     session[:user_id] = au_user.id
+    redirect_to root_url :notice => "Signed in!"
```

Comment:
What does this `:notice => "Signed in!"` do? It just seems to put something in the address bar / at the end of the URL.